### ISSUE TITLE
Make standalone app for Windows

### ISSUE THREAD

**Issue by harshsbais** [harshsbais/Amazon-Price-Drop-Alert#9]

**Comment by Amit366** [comment]
Sir standalone app means progressive webapp??

**Comment by Amit366** [comment]
I would like to work on this.

**Comment by harshsbais** [comment]
no not a webapp but an .exe app

**Comment by Amit366** [comment]
Sir I can solve it. Please assign it to me.

**Comment by harshsbais** [comment]
ok i'll assign to you but don't do it now. there's a already a pr in line which we would change the gui of the app.

**Comment by Amit366** [comment]
Sir, can I fork now?

**Comment by harshsbais** [comment]
yes sure you can fork now and start exploring things on how to do it but don't create the pr before the UI changes are done

**Comment by Amit366** [comment]
How can I install the dependencies??

**Comment by Amit366** [comment]
Sir I got some errors which I posted on discord. Please can you help me?

**Comment by Amit366** [comment]
Sir can I send the pull request??  I have sent the pic of the folders in the discord.

**Comment by Sobhit25** [comment]
Sir, I couldn't ask earlier as this PR was already taker up by @Amit366. 
May, I work on this issue with him if @Amit366 is happy with that.
I found this project interesting and in this way, I may be able to understand the code with him and think of some new features to add to it.

**Comment by harshsbais** [comment]
@Sobhit25 yes sure work on it
